### PR TITLE
feat(#1): The SPA shall have a button for uploading files

### LINKED ISSUE #2: The SPA shall have a button for uploading files.
The `SPA` shall `accept directories and the next MIME types`.

- image/jpeg
- image/gif
- image/png
- image/webp
- video/x-msvideo
- video/h264
- video/x-m4v
- video/mpeg
- video/mp4
- video/ogg
- video/webm

### PR DESCRIPTION
**Pull request by G2p1**
- Add upload button
- Add submit button

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -1,2 +1,22 @@
...
+            name="media_uploads" 
+            id="media_uploads">
+   
+        <button class="button is-primary is-outlined">Send</button>
+    </div>
+</div>
+</form>
+
+<style>
+    @import "https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma-rtl.min.css";
```
Comment (s1ckret) [inline src/routes/+page.svelte]: Please import bulma styles in one place. Also it should be global css, available for all files, like `src/app/app.css`. Or just link it in app.html.

File: src/css/app.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+@import "https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma-rtl.min.css";
+span{
+    color: red;
+}
+.button{
+    color:red;
+}
```
Comment (s1ckret) [inline src/css/app.css]: Move non-global styles into svelte component.

File: src/routes/+page.svelte [hunk 3]
```diff
@@ -1,2 +1,22 @@
-<h1>Welcome to SvelteKit</h1>
-<p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
+<script>
+
+</script>
+
+<form action="#">
```
Comment (s1ckret) [inline src/routes/+page.svelte]: Why do we need a form?
When should we use form and when a simple button will do? Can you please explore this?

**Review by s1ckret — CHANGES_REQUESTED** [review]

File: src/routes/+page.svelte (hunk 3 above)
Comment (G2p1) [reply src/routes/+page.svelte]: I will explore this, i thinked that we have a multiple files and we want to send them to server, so form is created for this reason ,isnt it?

Comment (G2p1) [reply src/routes/+page.svelte]: Yes, i read about it and agree with that the form not necessary

Comment (s1ckret) [reply src/routes/+page.svelte]: We send only request to a web server. After that request SPA will upload files directly to S3.

**Review by s1ckret — CHANGES_REQUESTED** [review]
Code looks good for me!
Please rename commits to the next format:
`feat(#1): your description`

Also squash your fix commits to `feat` one.

hint: `git rebase -i` (interactive rebase)

**Review by s1ckret — APPROVED** [review]

File: src/routes/+page.svelte (hunk 1 above)
Comment (G2p1) [reply src/routes/+page.svelte]: Done

File: src/css/app.css (hunk 2 above)
Comment (G2p1) [reply src/css/app.css]: Done

**Pull request merged**